client/overview/survey/context.tsx

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
nitpick: we could spilt the internal and external dependencies to structure the code better, now they seem to be mixed

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/8309/commits/2f32d7cf7f9d001384c5c61315e2c249ac863406 thank you!